### PR TITLE
Remove old, not so maintained and buggy baggy theme

### LINKED ISSUE #3460: Baggy theme deprecation
Maybe it is time to mark our long time loved baggy buddy as deprecated and invite users to switch to material. I don't know if you have any information about this theme being still used.
What do you think?

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | YES !
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | no
| Translation   | no
| CHANGELOG.md  | no
| License       | MIT

Bye, baggy. You were a good friend. 

I have a test who fails in testChangeReadingSpeed. Don't know why. 

Fix #3460

### CODE REVIEW

**Review by Kdecherf — DISMISSED** [review]
I don't think we should remove this theme like that.

I would prefer to deprecate it and stop supporting new features in it in the upcoming 2.4 version. And actually remove it in the next release.

**Comment by nicosomb** [comment]
@Kdecherf @tcitworld do you know why assets building fail?

**Comment by Kdecherf** [comment]
@nicosomb probably because you don't use the same versions of dependencies in charge of building assets than Github Actions.

Make sure you run `yarn install` before running `yarn build:prod`

**Comment by Kdecherf** [comment]
@wallabag/core as we will end with a unique theme, should we keep LiipTheme and other dependencies related to theme management?

**Comment by j0k3r** [comment]
I think so.
We know that maintaining many themes is complex.

**Comment by nicosomb** [comment]
I think that we can remove it.

**Comment by yguedidi** [comment]
@nicosomb @j0k3r @Kdecherf any news here?
investigating a potential upgrade to Twig 3, I see that liip/theme-bundle is a blocker and not maintained anymore.
any intention to preserve theming possibilities? or in addition to the Baggy theme removal we could also remove everything related to theming like this bundle?

**Comment by yguedidi** [comment]
just for info, in the end Twig 3 upgrade can be done only after Symfony 4 upgrade, as FOSUserBundle start to support it only starting from 2.2, which dropped support for Symfony 3.4

**Comment by Kdecherf** [comment]
@yguedidi I would go with theme selection removal, thus freeing us from liip/theme-bundle as well.

Any thought @wallabag/core?

**Comment by yguedidi** [comment]
@Kdecherf didn't know about the @wallabag/core team, but it seems I can't mention it and clicking on it leads to a 404 😅

File: web/wallassets/manifest.json [hunk 1]
```diff
@@ -63,14 +59,9 @@
...
   "public.css.map": "public.css.map",
   "public.js": "public.js",
-  "themes/_global/img/bg-select.png": "themes/_global/img/bg-select.png",
   "themes/_global/img/icons/carrot-icon--black.png": "themes/_global/img/icons/carrot-icon--black.png",
-  "themes/_global/img/icons/carrot-icon--white.png": "themes/_global/img/icons/carrot-icon--white.png",
-  "themes/_global/img/icons/Diaspora-asterisk.svg": "themes/_global/img/icons/Diaspora-asterisk.svg",
   "themes/_global/img/icons/diaspora-icon--black.png": "themes/_global/img/icons/diaspora-icon--black.png",
   "themes/_global/img/icons/scuttle.png": "themes/_global/img/icons/scuttle.png",
   "themes/_global/img/icons/shaarli.png": "themes/_global/img/icons/shaarli.png",
-  "themes/_global/img/icons/unmark-icon--black.png": "themes/_global/img/icons/unmark-icon--black.png",
```
Comment (Kdecherf) [inline web/wallassets/manifest.json:73]: `unmark-icon--black.png` seems also used by the material theme

**Review by Kdecherf — CHANGES_REQUESTED** [review]

File: web/wallassets/manifest.json (hunk 1 above)
Comment (nicosomb) [reply web/wallassets/manifest.json:73]: The line is available just below.

**Review by j0k3r — COMMENTED** [review]
One test it broken

**Comment by nicosomb** [comment]
Testsuite should be green now. 

And I fixed a visual bug thanks to that. 

Before 

Capture d’écran 2022-11-02 à 16 24 09

After

Capture d’écran 2022-11-02 à 16 24 19

**Comment by j0k3r** [comment]
@nicosomb it was a bug on the current 2.6.0 branch? 🤔

**Comment by nicosomb** [comment]
I don't know. I already it before my today rebase. I can't remember.

**Review by j0k3r — APPROVED** [review]

**Comment by j0k3r** [comment]
Yay conflicts again

**Review by j0k3r — APPROVED** [review]

**Pull request merged**